src/applications/facility-locator/components/SearchResultsHeader.jsx

```diff
@@ -18,7 +18,7 @@ const SearchResultsHeader = ({
     <h2
       id="facility-search-results"
       className="vads-u-font-family--sans vads-u-font-weight--normal vads-u-font-size--base vads-u-padding--0p5 vads-u-margin-y--1"
-      style={{ 'margin-left': '12px' }}
+      style={{ marginLeft: '12px' }}
```

Comment:
Was getting this warning on a dev build with the previous style.
> Warning: Unsupported style property margin-left. Did you mean marginLeft?

Reply:
For reasons I can't explain, I could've sworn the header wasn't rendering without this at some point, but now I can't reproduce it. In any case, I just left this fix in. You can verify that it shows a warning on dev builds by searching anything in the app on dev.

Reply:
Yeah thanks for fixing that. I noticed it myself as well and was going to push up a fix in my next PR.